gitlabform/processors/__init__.py

```diff
@@ -28,21 +28,10 @@ def process_entity(
     ):
         for processor in self.processors:
             if only_sections == "all" or processor.configuration_name in only_sections:
-                try:
```

Comment:
This part of the PR just restores the code before by debugging that accidentally got pushed to `main`. It's not related with the rest of the PR.